client/src/Login/Login.js

```diff
@@ -43,9 +40,7 @@ export default function Login() {
             .then(function (response) {
```

Comment:
use anonymous like this : .then((response) => { rest of the function }

Reply:
Done